package.json

```diff
@@ -21,10 +21,10 @@
     "touch": "0.0.3"
   },
   "devDependencies": {
-    "coffee-script-redux": "2.0.0-beta7",
-    "mocha": "~1.12.1"
+    "coffee-script": "^1.9.1"
   },
   "scripts": {
-    "test": "echo 'write some tests'"
+    "build": "./node_modules/.bin/coffee -cbo lib src",
```

Comment:
package.json scripts have node_modules/.bin at the front of their PATH by default - just `coffee` is enough.